auditlog/README.rst

```diff
@@ -11,6 +11,11 @@ Go to `Reporting / Audit / Rules` to subscribe rules. A rule defines which
 operations to log for a given data model.
```

Comment:
Hi @sebalix Thanks for the contribution.
I miss the top of the README file: badge + line for document title.